vmdb/app/views/layouts/_edit_to_email.html.haml

```diff
@@ -12,9 +12,11 @@
             = _("None")
           - else
             - if @temp[:email_to].present?
-              - emails = @temp[:email_to].sort { |a, b| a[1] <=> b[1] }.collect { |e| e[0] }
+              - emails = @temp[:email_to]
+                  .sort_by { |_email, display_email| display_name }
+                  .collect { |email, _display_email| email }
             - elsif !@edit[:new][:email][:to].present?
-              - emails = @edit[:new][:email][:to].sort { |a, b| a[1] <=> b[1] }
+              - emails = @edit[:new][:email][:to].sort
```

Comment:
I presume the original code assumed `@edit[:new][:email][:to]` was key-value pairs of email-display_email, much like the previous condition, but in researching the code, I don't think this is correct.

However, I think there's another bug here.  This branch checks if it's `!present?` (i.e. `blank?`) and THEN sorts it.  That's a straight-up bug.  @romanblanco @skateman could this be a haml conversion bug?